Clean radium apps (#24)
STORM-IRIT/Radium-Apps: README.md

Radium is a research 3D Engine for rendering, animation and processing.
It is developed and maintained by the [STORM research group](https://www.irit.fr/STORM/site/).

This repository holds front-end applications (GUI, command line) based on Radium Libraries
 - Sandbox: Graphical frontend of the Radium Engine. This application aims at demonstrating the capabilities of the Radium Libraries. It can be extended using Radium plugins.
 - CLISubdivider: Example of command line application. Loads obj files and run subdivision algorithms implemented with OpenMesh.
 
To get more details about each application, checkout the Readme files in each application directory.

# How to get these applications
## Pre-compiled binaries

> Pre-compiled binaries can be compiled at https://github.com/STORM-IRIT/Radium-Releases/releases/.

Reviewer: ```suggestion
Pre-compiled binaries can be downloaded from https://github.com/STORM-IRIT/Radium-Releases/releases/.
```

> It includes:

Reviewer: ```suggestion
The downloaded archive contains:
```

 - Radium libraries

>  - Radium applications

Reviewer: ```suggestion
 - Radium exemple applications
     - A command line interface demo
     - A full featured graphical application
```

## Compilation instructions

> To compile this repository, you first need to compile the [Radium libraries](https://github.com/STORM-IRIT/Radium-Engine).

Reviewer: ```suggestion
To compile this repository, you first need to compile and install the [Radium libraries](https://github.com/STORM-IRIT/Radium-Engine).
```

> Then, configure this project using cmake. You need to give the location of Radium package by setting `Radium_DIR` (see more details in Radium Engine documentation: https://storm-irit.github.io/Radium-Engine/).

Reviewer: ```suggestion
Then, configure this project using cmake. 
You need to tell cmake where to find the installed Radium package by setting the `Radium_DIR` variable to the right location (see more details in Radium Engine documentation: https://storm-irit.github.io/Radium-Engine/).
```
